### ISSUE TITLE
Feature request? Volume control for transmit and say commands

### ISSUE THREAD

**Issue by rodan123** [EDCD/EDDI#1921]
## EDDI version in which issue is found
3.7.0

## VoiceAttack version in which issue is found (as applicable)
1.8.6

## Steps to reproduce
1. Call EDDI plugin from VA with context "say" or "transmit" 

## Expected
A way to set the volume of the speech played.

## Observed
Cannot find a way to pass volume level to the plugin.

## Investigation
Is this possible now? I cannot pass SSML tags to these functions.  (prosody volume)
Is there a way to set the volume for a particular function call?
If not could a parameter be added, similar to setting the "Voice" ?

Thanks.

**Comment by rodan123** [comment]
I whipped up some code to implement this feature in my build. 
If you like I can create a pull request, unless you're already working on an independent solution.

**Comment by Tkael** [comment]
Does the current `SpeechVolume()` Cottle method meet your needs? It is possible to pass escaped text to EDDI through the plugin (using `|` as an escape character) and have it render using any of the current custom functions. 

If not, what approach did you have in mind?

**Comment by rodan123** [comment]
I attached PR #1930.

I use the EDDI 'transmit' function with VA to distort and playback E:D messages from players and NPCs, along with radio sound effects, and a script to assign a gendered voice based on the CMDR name. I found that the voice volume was too loud and in populated systems with a lot of chatter, it gets annoying. So, I was looking to control the voice volume. I suppose I could rewrite it all in cottle and use `SpeechVolume()`, as suggested,  but what I have works well and changing EDDI is easier. 

Feel free to delete the PR, if you think it won't be useful to the general user base. I can continue with it personally since I run a custom build of EDDI.
